db/migrate/20160219143618_create_questions.rb

```diff
@@ -0,0 +1,9 @@
+class CreateQuestions < ActiveRecord::Migration
+  def change
+    create_table :questions do |t|
+      t.string :question_text, null: false
```

Comment:
Will the question text ever be longer than 256 characters? If so this should be datatype text. `t.text`